### DISCUSSION TITLE
2024/adventures/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/shane0.github.io#54] [General]
# 2024/adventures/

http://shanenull.com/2024/adventures/

**Comment by shane0** [comment]
# adventures

**Reply by shane0** [reply]
- on adventures I often listen to cormac mccarthy audiobooks
- the desert trilogy in the desert or blood meridian

**Reply by shane0** [reply]
# 2025 bicycle pnw pacific coast

## posts

- will be on instagram & facebook might create channels for this trip

## prep

- aca maps
- test rides & camps
- to reverse routes:  long press the "Back" button, navigate to the "Navigation" option, and then select "Back to Start
- hmg mid 2 & tarp
- soma wolverine build w/ rohloff ergon fully loaded for winter

**Reply by shane0** [reply]
https://shanenull.com/2024/pacific_coast/

**Comment by shane0** [comment]
# what is a real friend

“He gives what is beautiful,
hard to give;
does what is hard to do;
endures painful, ill-spoken words.
His secrets he tells you;
your secrets he keeps.

When misfortunes strike,
he doesn’t abandon you;
when you’re down & out,
doesn’t look down on you.
A person in whom these traits are found,
is a friend to be cultivated
by anyone wanting a friend.”

https://www.dhammatalks.org/suttas/AN/AN7_35.html

**Reply by shane0** [reply]
“If you want to go fast, go alone. If you want to go far, go together.”

**Reply by shane0** [reply]
"It is no measure of health to be well adjusted to a profoundly sick society." - Jiddu Krishnamurti

**Reply by shane0** [reply]
# inspiration

- for cold days or physical crash

If you can keep your head when all about you
Are losing theirs and blaming it on you,
If you can trust yourself when all men doubt you,
But make allowance for their doubting too;
If you can wait and not be tired by waiting,
Or being lied about, don’t deal in lying,
Or being hated, and still hating not,
Or being wronged, and still forgiving;

If you can keep your head when all about you
Are losing theirs and blaming it on you,
If you can trust yourself when all men doubt you,
But make allowance for their doubting too;
If you can wait and not be tired by waiting,
Or being lied about, don’t deal in lying,
Or being hated, and still hating not,
Or being wronged, and still forgiving;

If you can dream—and not make dreams your master;
If you can think—and not make thoughts your aim;
If you can meet with Triumph and Disaster
And treat those two impostors just the same;
If you can bear to hear the truth you’ve spoken
Twisted by knaves to make a trap for fools,
Or watch the things you’ve built with greatest care
Destroyed by fools, and build them up again;

If you can keep your head when all about you
Are losing theirs and blaming it on you,
If you can trust yourself when all men doubt you,
But make allowance for their doubting too;
If you can wait and not be tired by waiting,
Or being lied about, don’t deal in lying,
Or being hated, and still hating not,
Or being wronged, and still forgiving;

If you can keep your head when all about you
Are losing theirs and blaming it on you,
If you can trust yourself when all men doubt you,
But make allowance for their doubting too;
If you can wait and not be tired by waiting,
Or being lied about, don’t deal in lying,
Or being hated, and still hating not,
Or being wronged, and still forgiving;

If you can talk with crowds and keep your virtue,
Or walk with Kings—nor lose the common touch,
If neither foes nor loving friends can hurt you,
If all men count with you but none too much,
If you can fill the unforgiving minute
With sixty seconds’ worth of distance run,
Yours is the Earth and everything that’s in it,
And—which is more—you’ll be a Man, my son!

**Comment by shane0** [comment]
https://www.pnt.org/

**Comment by shane0** [comment]
# davis tent

```text
5′ walls in front sloping to 2′ 5″ wall in back
Zippered D Door
Attached Sod Cloth to three sides (seals tent to the ground)
Attached 4′ 10″  Awning
Ropes cut to size with metal tensioners installed
18 Steel Stakes in their own bag
Tent weight = 18lbs
Traditional pole weight with bag= 7lbs
```

**Reply by shane0** [reply]
https://www.davistent.com/product/single-shot-canvas-tent/

**Reply by shane0** [reply]
test
https://www.youtube.com/watch?v=SBNYF8n3wGY&t=38s

**Comment by shane0** [comment]
https://pctmap.net/2014/03/04/elevation-gain-on-the-pacific-crest-trail/

Sure, here are tables for the Washington, Oregon, and California sections of the Pacific Crest Trail, summarizing the elevation gain and loss for each section:

**Washington:**

| Location             | Elevation Gain (ft) | Elevation Loss (ft) |
|----------------------|---------------------|----------------------|
| Columbia River to Hwy 12 | 36,196              | 35,500               |
| Hwy 12 to White Pass | 27,144              | 27,201               |
| White Pass to Snoqualmie Pass | 27,265              | 27,378               |
| Snoqualmie Pass to Stevens Pass | 29,070              | 28,560               |
| Stevens Pass to Rainy Pass | 37,303              | 36,241               |
| Rainy Pass to Manning Park | 36,563              | 36,791               |
| **Total**            | **193,541**         | **191,671**          |

**Oregon:**

| Location               | Elevation Gain (ft) | Elevation Loss (ft) |
|------------------------|---------------------|----------------------|
| Columbia River to HWY 35 | 33,306              | 29,799               |
| HWY 35 to Ollalie Lake  | 24,195              | 24,025               |
| Ollalie Lake to Hwy 242 | 21,352              | 20,526               |
| Hwy 242 to McKenzie Pass | 19,762              | 19,542               |
| McKenzie Pass to Hwy 126 | 20,186              | 19,944               |
| Hwy 126 to Hwy 58       | 20,782              | 21,398               |
| Hwy 58 to Shelter Cove  | 20,455              | 20,594               |
| Shelter Cove to Crater Lake | 24,131              | 23,968               |
| Crater Lake to Hwy 138  | 24,568              | 23,961               |
| Hwy 138 to Hwy 140      | 17,848              | 17,798               |
| Hwy 140 to Hwy 66       | 23,804              | 24,547               |
| Hwy 66 to Callahan's    | 25,482              | 26,396               |
| **Total**              | **275,071**         | **271,388**          |

**California:**

| Location                     | Elevation Gain (ft) | Elevation Loss (ft) |
|------------------------------|---------------------|----------------------|
| Callahan's to Seiad Valley  | 30,793              | 33,002               |
| Seiad Valley to Etna Summit | 23,013              | 22,844               |
| Etna Summit to Castella      | 38,398              | 38,183               |
| Castella to Burney Falls     | 21,283              | 21,173               |
| Burney Falls to Old Station  | 29,151              | 27,769               |
| Old Station to Hwy 36        | 30,538              | 30,469               |
| Hwy 36 to Hwy 44             | 31,144              | 29,705               |
| Hwy 44 to Hwy 36             | 29,681              | 30,280               |
| Hwy 36 to I-5                | 26,849              | 25,704               |
| I-5 to Donner Pass          | 50,990              | 50,143               |
| Donner Pass to Echo Lake     | 31,927              | 30,759               |
| Echo Lake to Carson Pass     | 20,229              | 19,568               |
| Carson Pass to Ebbetts Pass  | 24,170              | 24,319               |
| Ebbetts Pass to Sonora Pass  | 25,946              | 25,299               |
| Sonora Pass to Echo Lake     | 30,360              | 29,674               |
| Echo Lake to Highway 50      | 32,833              | 32,774               |
| Highway 50 to Barker Pass    | 22,982              | 22,774               |
| Barker Pass to Highway 80    | 30,743              | 29,977               |
| Highway 80 to Sierra City    | 31,627              | 31,065               |
| Sierra City to Belden        | 27,973              | 27,721               |
| Belden to Highway 36         | 34,201              | 34,089               |
| Highway 36 to Highway 44     | 23,478              | 22,968               |
| Highway 44 to Highway 36     | 30,659              | 30,488               |
| **Total**                    | **621,556**         | **614,956**          |

These tables provide a breakdown of the elevation gain and loss for each section of the Pacific Crest Trail in Washington, Oregon, and California, with a final row for the total gain and loss in each state.

**Comment by shane0** [comment]
# overland adventures

- Jeep tent setup
- https://youtu.be/SBNYF8n3wGY?si=46odWXkKUESfNC6g

**Comment by shane0** [comment]
# motorcycle camping

- motorcycled seattle to colorado springs to a friends then to wisconsin to visit my father
- https://youtu.be/VyoAHX52QsI
- https://youtu.be/ndScAtFGuJw

**Comment by shane0** [comment]
# emergency

- I usually carry an ifak & a trauma kit for emergencies
- I avoid knives & fires unless I can quickly access healthcare facilities
- if using wood use a saw vs an axe
- threats from animals including humans: noise such as horns & trip alarms or bear spray pepper spray LTL firearms & last resort most threats are avoided with common sense do not camp in stupid places
- do not go fast if bicycling or motorcycling its not a race
- pace yourself
- eat or sleep
- let people know where you are and where you are going
- learn to use a compass gps paper maps the sun and stars
- solar distillery works with plants or urine
- do not eat plants you are not %100 certain are edible
